packages/neuron-ui/src/utils/formatters.ts

```diff
@@ -295,6 +297,43 @@ export const nftFormatter = (hex?: string, idOnly = false) => {
   return `#${id} mNFT`
 }
 
+export const sporeFormatter = (args?: string, data?: string, name?: string) => {
+  let format = 'Spore'
+
+  const SporeData = molecule.table(
+    {
+      contentType: blockchain.Bytes,
+      content: blockchain.Bytes,
+      clusterId: blockchain.BytesOpt,
+    },
+    ['contentType', 'content', 'clusterId']
+  )
+
+  if (!data) {
+    format = `[Unbound] ${format}`
```

Comment:
Should `Unbound` adapt i18n?

Reply:
> Should `Unbound` adapt i18n?

@Danie0918 do you have any idea about this?

Reply:
Is this a product-related issue? I didn't quite understand the current situation.

Reply:
> Is this a product-related issue? I didn't quite understand the current situation.

<img width="408" alt="image" src="https://github.com/nervosnetwork/neuron/assets/7511174/2da402e5-79cc-4ba7-bccf-b43d1ce9a9de">

An **unbound** spore is not derived from a cluster. We should consider how to handle an unbound mark in i18n or leave it here.

Reply:
Here we don't show [Unbound], only the ID and Spore.

Reply:
> Here we don't show [Unbound], only the ID and Spore.

Okay, I misunderstood the PRD, I'll hide the cluster name if the spore isn't derived from a cluster

Reply:
I've removed the unbound mark for the spores that aren't derived from a cluster